exchange/exchange.go

```diff
@@ -1247,12 +1247,12 @@ func (e *exchange) makeBid(bids []*entities.PbsOrtbBid, auc *auction, returnCrea
 	for _, bid := range bids {
 		if !dsa.Validate(bidRequest, bid) {
 			RequiredError := openrtb_ext.ExtBidderMessage{
-				Code:    errortypes.BadServerResponseErrorCode,
-				Message: "bidResponse rejected: DSA object missing when required",
+				Code:    errortypes.InvalidBidResponseDSAWarningCode,
+				Message: "bid response rejected: object missing when required",
```

Comment:
Why did you remove DSA from the message?

Reply:
I figured the code gives the context. Do you want me to add it back?

Reply:
Discussed offline. Added it back for human readability.